### PR TITLE
Fetch slack file info if we are missing a mimetype

### PR DESCRIPTION
**Pull request by Half-Shot**
As a workaround for #432

### CODE REVIEW

File: src/BridgedRoom.ts [hunk 1]
```diff
@@ -657,6 +657,102 @@ export class BridgedRoom {
...
+                    uri: file.url_private!,
+                });
+            } catch (ex) {
+                log.error("Failed to download snippet", ex);
+                return;
+            }
+            let htmlCode = "";
+            // Because escaping 6 backticks is not good for readability.
+            // tslint:disable-next-line: prefer-template
+            const code = "```" + `\n${htmlString}\n` + "```";
```
Comment (jaller94) [inline src/BridgedRoom.ts:684]: Is there a way to escape ``` if it was part of htmlString?

File: src/BridgedRoom.ts [hunk 2]
```diff
@@ -657,6 +657,102 @@ export class BridgedRoom {
...
+            htmlCode += substitutions.htmlEscape(htmlString);
+            htmlCode += "</code></pre>";
+
+            const messageContent = {
+                body: code,
+                format: "org.matrix.custom.html",
+                formatted_body: htmlCode,
+                msgtype: "m.text",
+            };
+            await ghost.sendMessage(this.matrixRoomId, messageContent, channelId, slackEventId);
```
Comment (jaller94) [inline src/BridgedRoom.ts:699]: We don't seem to check or handle a possible rejection due to the snippet possibly being too long.
I'm not asking you to implement this now, but file it, if you think that's a valid feature request that will come up.

Comment (Half-Shot) [reply src/BridgedRoom.ts:699]: Good point, we should upload as a file if it's more than >~64Kb

Comment (Half-Shot) [reply src/BridgedRoom.ts:699]: Created issue #450 to track.

File: src/BridgedRoom.ts (hunk 1 above)
Comment (Half-Shot) [reply src/BridgedRoom.ts:684]: Opened #461

File: src/BridgedRoom.ts (hunk 2 above)
Comment (Half-Shot) [reply src/BridgedRoom.ts:699]: Opened #460

**Pull request merged**